scripts/cve/create_cve_trackers.py

```
@@ -0,0 +1,566 @@
+    try:
+        import certifi
+        ctx.load_verify_locations(certifi.where())
+    except ImportError:
+        # On macOS, try to use the system certificates
+        import subprocess
+        import tempfile
+        try:
+            # Try to get certificates from security command on macOS
+            result = subprocess.run(
+                ["security", "find-certificate", "-a", "-p",
+                 "/System/Library/Keychains/SystemRootCertificates.keychain"],
```

Ruff (0.15.0)

[error] Starting a process with a partial executable path

(S607)